pages/_app.js

```diff
@@ -1,4 +1,6 @@
 import '../styles/globals.css';
+import '../styles/datepicker.css';
+// import 'rsuite/dist/rsuite.min.css';
```

Comment:
Can we try to use the CSS from `import 'rsuite/dist/rsuite.min.css' directly instead of copying it over to datepicker.css. The advantages of this is that it will enable us to update the datepicker library without caring for the CSS as the CSS we are using will be automatically updated.

Reply:
`<a href=""></a>` link inside the date picker css override  over `globalstyles css` for `<a href=""></a>`

Reply:
As discussed, can we try placing the rsuite CSS above our CSS. Not sure if it would work, but it is worth a trial.